### ISSUE TITLE
You spawn with a Starting House even when the config is set to Nothing

### ISSUE THREAD

**Issue by aaronhowser1** [Brian-Wuest/MC-Prefab#164]
1. What version of Minecraft, Forge and Mod are you running?
    1. Minecraft Version: 1.16.5
    1. Forge Version: 36.1.4
    1. Mod Version:prefab-1.6.5.0
1. What type of issue is this? Choose one of the following:
    1. **Bug** 
```
[general]
	#Determines which starting item a player gets on first world join. Valid values for this option are: "Starting House", "Moderate House", "Structure Part", "Nothing". Server configuration overrides client.
	"Starting Item" = "Nothing"
```
This is in the defaultconfig, and it does get carried over to new worlds. However, I still spawn with a Starting House.

**Comment by Brian-Wuest** [comment]
I'm sorry but I was not able to duplicate this issue.
I was running a local modpack through CurseForge desktop with just a few mods (Like JEI) and Prefab.

Local Modpack setup:
![image](https://user-images.githubusercontent.com/13502068/114169990-e7e7bc00-98f7-11eb-8e64-0056c5070087.png)


I updated my setting to be "Nothing" and created a new world and did not receive a starting item (as expected).

**Something to note**: If you launched the game; then changed the configuration file; I think it will still hold onto the old value. Forge hasn't gotten around to properly re-implementing it's configuration file system yet after they removed it when they did a big revamp of their codebase for 1.13+. So this should work properly if you close and re-launch minecraft after making your changes.

Here is a screen shot of my configuration file as well as my log file.
In the log file there will be a message about "[PLAYER_NAME] joined the game for the first time. Giving them starting item.". If you don't see that message then this is working properly.

![image](https://user-images.githubusercontent.com/13502068/114168870-765b3e00-98f6-11eb-9f6a-74b6bc9740c1.png)

Additionally; you may want to review the "fml.toml" configuration file to see if the "defaultConfigPath" has been updated/changed. It should look like this:

![image](https://user-images.githubusercontent.com/13502068/114169631-6bed7400-98f7-11eb-9839-4547c65ec8f2.png)

**Comment by aaronhowser1** [comment]
> Something to note: If you launched the game; then changed the configuration file; I think it will still hold onto the old value. Forge hasn't gotten around to properly re-implementing it's configuration file system yet after they removed it when they did a big revamp of their codebase for 1.13+. So this should work properly if you close and re-launch minecraft after making your changes.

That was probably it, sorry 😅

**Comment by aaronhowser1** [comment]
Ok no, restart didn't fix it.

**Comment by Brian-Wuest** [comment]
Can you please provide your log files.
In addition; are you playing on single player or on a server?

**Comment by aaronhowser1** [comment]
```toml
[general]
	#Determines which starting item a player gets on first world join. Valid values for this option are: "Starting House", "Moderate House", "Structure Part", "Nothing". Server configuration overrides client.
	"Starting Item" = "Nothing"
```
This is the world's config, brand new and unchanged. It did correctly load the change from the `defaultconfigs` folder (which isn't changed in fml.toml), since the normal default is `"Starting House"`. 

I'm on single player, here's the logs (too big for pastebin, sorry):
[latest.log](https://github.com/Brian-Wuest/MC-Prefab/files/6295914/latest.log)
[debug.log](https://github.com/Brian-Wuest/MC-Prefab/files/6295915/debug.log)

**Comment by Brian-Wuest** [comment]
Thank you for the logs, looking through them it looks like there is a "serverconfig" folder.

![image](https://user-images.githubusercontent.com/13502068/114383560-4a88c400-9b53-11eb-8de6-5ac6cde23a47.png)


Can you check this location to see if that's where the starting house is coming from?

**Comment by aaronhowser1** [comment]
The serverconfig folder exists in every world, when the world is created it copies whatever's in defaultconfigs and puts it there.

Both files are identical, and both have it set to `"Nothing"`

**Comment by Brian-Wuest** [comment]
Is this a public modpack? I want to pull it local and see if I can duplicate it.

**Comment by aaronhowser1** [comment]
It's barely starting dev now. I can send over an export though I suppose. 
~~FTB University 1.16 dev-dev.zip~~
Obviously please don't send this to anyone ;)

**Comment by Brian-Wuest** [comment]
Thanks, I will take a look at this when I am done with work!

**Comment by Brian-Wuest** [comment]
I got the file, @aaronhowser1  feel free to remove the link

**Comment by Brian-Wuest** [comment]
Got the issue figured out in development. Double checking with the modpack provided.

**Comment by Brian-Wuest** [comment]
Got the issue fixed with b386c1a.

I will be pushing an update to CurseForge shortly.

This update will also fix an issue with the Tree Nether gate containing a Wither Rose.

**Comment by Brian-Wuest** [comment]
This has been released to CurseForge for version 1.16.5.1.

This will become available as soon as it's approved.

Thank you for finding this issue your patience with this issue.

**Comment by aaronhowser1** [comment]
Awesome! Thanks for the quick fix
